cohort-engine/src/main/java/com/ibm/cohort/engine/measure/MeasureEvaluator.java

```diff
@@ -172,7 +193,25 @@ public MeasureReport evaluatePatientMeasure(Measure measure, String patientId, S
 			}
 		}
 
+		measure.getExtension().stream()
+				.filter(this::isParameterExtension)
+				.forEach(measureDefault ->
+						         seed.getContext().setParameter(
+								         null,
+								         measureDefault.getId(),
+								         toCqlObject(measureDefault.getValue())));
+
```

Comment:
I think this is in the wrong place. This should occur _before_ the part were we set the parameters based on the parameters map provided in method invocation. The map is the override values. We want those applied after the defaults specified in the Measure resource.

Reply:
ah, yep. good call.